banzai_floyds/tests/test_wavelengths.py

```diff
@@ -106,21 +106,47 @@ def test_correlate_peaks():
 def test_refine_peak_centers():
     # use well-behaved seed
     seed = 75827
-    line_width = 3
+    line_sigma = 3
     line_sep = 10
-    input_spectrum, lines, test_lines = build_random_spectrum(seed=seed, line_sigma=line_width)
+    input_spectrum, lines, test_lines = build_random_spectrum(seed=seed, line_sigma=line_sigma)
 
     recovered_peaks = identify_peaks(input_spectrum, 0.01 * np.ones_like(input_spectrum),
-                                     sigma_to_fwhm(line_width), line_sep)
+                                     sigma_to_fwhm(line_sigma), line_sep)
 
     fit_list = refine_peak_centers(input_spectrum, 0.01 * np.ones_like(input_spectrum),
-                                   recovered_peaks, sigma_to_fwhm(line_width))
+                                   recovered_peaks, sigma_to_fwhm(line_sigma))
 
     # Need to figure out how to handle blurred lines and overlapping peaks.
     for fit in fit_list:
         assert np.min(abs(test_lines - fit)) < 0.2
 
 
```

Comment:
Previously our unit test didn't include noise and worked fine. Once we got noisy data, we discovered a bug so this tests against that regression.